src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildACppCliProject.cs

```diff
@@ -52,7 +57,27 @@ public void It_builds_and_runs()
                 .HaveStdOutContaining("Hello, World!");
         }
 
-        [FullMSBuildOnlyFact(Skip = "https://github.com/dotnet/sdk/issues/3785")]
+        [FullMSBuildOnlyFact]
+        public void It_builds_and_runs_with_package_reference()
+        {
+            var targetFramework = ToolsetInfo.CurrentTargetFramework + "-windows";
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("NetCoreCsharpAppReferenceCppCliLib")
+                .WithSource()
+                .WithProjectChanges((projectPath, project) => ConfigureProject(projectPath, project, "NewtonSoft.Json","13.0.1", targetFramework, new string[] { "_EnablePackageReferencesInVCProjects" , "IncludeWindowsSDKRefFrameworkReferences" }));
+
+            // build projects separately with BuildProjectReferences=false to simulate VS build behavior
```

Comment:
Is this comment supposed to be here?  I don't see anything setting BuildProjectReferences to false.